docs/configuration.rst

```diff
@@ -281,20 +95,20 @@ Removes a given reference set from a given data repository.
 
 .. code-block:: bash
 
-    $ ga4gh_repo remove-referenceset path/to/datarepo aReferenceSet
+    $ ga4gh_repo remove-referenceset path/to/repo.db aReferenceSet
 
 ++++++++++++++++
-add-ontologymap
+add-ontology
```

Comment:
As discussed in https://github.com/ga4gh/server/pull/1117, and in preparation for #1147 we want ``Ontology``.